medical-hub/src/Pages/Cart.jsx

```diff
@@ -1,306 +0,0 @@
-import { Box, Radio, Text, Image, Select, Button, useToast } from "@chakra-ui/react";
-import React, { useEffect, useState } from "react";
-import { Link, useParams } from "react-router-dom";
-
-
-import { ChevronDownIcon, AddIcon } from "@chakra-ui/icons";
-import { useDispatch, useSelector } from "react-redux";
-import { cartData } from "../redux/ProductAction"
-
-import axios from "axios";
-import CartCarousel from "./CartCarousel";
-
-const Cart = () => {
-  const Toast=useToast();
-  const cart_Data = useSelector((state) => state.cart);
-  console.log(cart_Data, "cartData");
-
-  const [qty, setQty] = useState(1);
-  const dispatch = useDispatch();
-  const params = useParams();
-  console.log(params);
-
-  var bag = 0;
-  for (var i = 0; i < cart_Data.length; i++) {
```

Comment:
Don't use for loops and if statements directly. Use it inside any lifecycle method or anything.